src/util.h

```diff
@@ -182,6 +182,7 @@ void ReleaseDirectoryLocks();
 
 bool TryCreateDirectories(const fs::path& p);
 fs::path GetDefaultDataDir();
+const fs::path &GetBlocksDir(bool fNetSpecific = true);
```

Comment:
I am wondering why this should be net specific, considering that we have the same concept for wallets and the `GetWalletDir` is not net specific.

Reply:
Isn't the blocks directory *by definition* net specific?